multinet/api.py

```diff
@@ -35,11 +37,13 @@ def get_workspace(workspace: str) -> Any:
 
 
 @bp.route("/workspaces/<workspace>/tables", methods=["GET"])
-@use_kwargs({"type": fields.Str()})
+@use_args({"type": fields.Str()})
 @swag_from("swagger/workspace_tables.yaml")
-def get_workspace_tables(workspace: str, type: TableType = "all") -> Any:
+def get_workspace_tables(args: Dict, workspace: str) -> Any:
     """Retrieve the tables of a single workspace."""
-    tables = db.workspace_tables(workspace, type)
+    # TODO: Fix mypy complaint about below expression.
+    table_type: TableType = args.get("type") or "all"  # type: ignore
+    tables = db.workspace_tables(workspace, table_type)
```

Comment:
Can you explain why this change is positive? It changes the signature of the function in a way that makes it less legible, IMO.

Reply:
I will mention, I understand why `type` is suggested to be changed, but the problem is that we're using `type` as a parameter for a public API function, and changing it to a non-keyword just to satisfy flake8 seems like a loss if it impacts the design of the API.

Reply:
The only reason that's it's positive is not having a variable named `type`. I agree it's a nuisance to  change all of this just for that. Ideally, we'd have a way to map the "type" key to a different variable name. I'd be okay adding an ignore to the type checker here though.

Reply:
Ok, let's put this function back to how it was and add an ignore directive (with a comment explaining why).